x-pack/test/functional/services/ml/data_frame_analytics_results.ts

```diff
@@ -51,6 +53,10 @@ export function MachineLearningDataFrameAnalyticsResultsProvider(
       await testSubjects.existOrFail('mlDFExpandableSection-results');
     },
 
+    resultsTableSelector(subSelector?: string) {
```

Comment:
It seems this isn't used at the moment, although we have a couple methods in this file that could make use of it. I think it makes sense to have this method so I'd suggest to update our usage of `~mlExplorationDataGrid > SOMETHING` to instead use `resultsTableSelector(SOMETHING)`.